css/main.css

```diff
@@ -695,6 +695,48 @@ footer {
   background-color: var(--color-vivid-pink);
 }
 
+.map-title {
+  color: var(--color-text-secondary);
+  margin: 50px 20px 32px;
+  text-align: center;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 150%;
+}
+
+.map-box {
+  height: 300px;
+  overflow: hidden;
+  border: 3px solid #e20665;
+  margin: 2% 10%;
+}
+
+#map {
+  overflow: visible;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  height: 300px;
+}
+
+.mapboxgl-canvas {
+  width: 100% !important;
```

Comment:
Is there any other to achieve the same without using `!important`?

Reply:
In the Mapbox library, in this canvas element, they have used inline CSS for width, so other than using !important, I didn't find anything to overwrite that style.

<img width="684" alt="image" src="https://user-images.githubusercontent.com/54801934/194915190-c401c6d2-f4b4-486a-b464-bb4a05b13933.png">

If you know, of any other way by which we can change the width, then please let me know.

Reply:
Got it, thanks for explanation, resolving the conversation

Reply:
Do Mapbox library not provide the CSS customization??
Also, why do you exactly want to change the width? What is the purpose here?

Reply:
After updating css, not needed this anymore, removed it